packages/driver/src/cypress/stack_utils.ts

```diff
@@ -390,7 +396,35 @@ const normalizedStack = (err) => {
   // Chromium-based errors do, so we normalize them so that the stack
   // always includes the name/message
   const errString = err.toString()
-  const errStack = err.stack || ''
+  let errStack = err.stack || ''
+
+  if (Cypress.isBrowser('webkit')) {
+    // WebKit will not determine the proper stack trace for an error, with stack entries
+    // missing function names, call locations, or both. This is due to a number of documented
+    // issues with WebKit:
+    // https://bugs.webkit.org/show_bug.cgi?id=86493
+    // https://bugs.webkit.org/show_bug.cgi?id=243668
+    // https://bugs.webkit.org/show_bug.cgi?id=174380
+    //
+    // We update these stack entries with placeholder names/locations to more closely align
+    // the output with other browsers, minimizing the visual impact to the stack traces we render
+    // within the command log and console and ensuring that the stacks can be identified within
+    // and parsed out of test snapshots that include them.
+    errStack = errStack.replaceAll(webkitStackLineRegex, (match, ...parts: string[]) => {
+      // We patch WebKit's Error within the AUT as CyWebKitError, causing it to
+      // be presented within the stack. If we detect it within the stack, we remove it.
+      if (parts[0] === '__CyWebKitError') {
```

Comment:
Is this because it's showing up in the stack trace of every `new Error`? Should we just not name the function then?

Reply:
I went back and forth on this. If we don't name it, it'll still render here, just as yet another anonymous '<unknown>' function. Given the frequency with which it'll be presented (for any uncaught exception thrown by the AUT), I figured its presence in the stacks might be confusing, named or not. But I could understand the argument that removing it is an improper obfuscation of what is actually being executed?

Reply:
hmm, yeah... maybe we just oughtta name it `Error`, then. This seems fine too. Not too big an issue for experimental.